grug/app/src/app.rs

```diff
@@ -1,12 +1,15 @@
+#[cfg(feature = "tracing")]
+use tracing::error;
```

Comment:
Instead of importing this, let's always invoke tracing macros as

```rust
tracing::error!(/* ... */);
```

Because just `error!` isn't very clear what it means. It can be misunderstood as the function returning an error. With `tracing::error!` it's clear.